### PR TITLE
Adventure Rebalance

### PR DESCRIPTION
**Pull request by zumbak04**
# Changelog:
- Balanced chance of raiding adventure spawn on vanilla (about 6 adventures in 10 years during first 50 ingame years, about 29 in 50-150 ingame years).
**_Note for Testers:_** Type in console `print_global_variables` to see `global_raiding_adventurer_spawn_by_courtier_MTTH` (number of spawned raiding adventures).
- Decreased chance of worgen pack spawn (about 2 packs in 10 years).
_**Note for Testers:**_ Type in console `print_global_variables` to see `global_worgen_pack_spawn` (number of spawned worgen packs).
- Decreased chance of sea ravager spawn (about 10 ravagers in 10 years).
**_Note for Testers:_** Type in console `print_global_variables` to see `global_sea_invader_spawn` (number of spawned sea ravagers).
- Among all the other sea ravagers, the naga are more likely to spawn after their resurfacing.
# Secret Changelog:
- Only players can get event to spawn annihilan adventurer. This should make adventures more balanceable.

### CODE REVIEW

**Comment by zumbak04** [comment]
@DukeJeans would like to hear what @MemeWizard will get:
My results in 583-603 testrun:
- **Global Raiding Variable: 3 - 10 yr average of 1.5**
- **Global Sea Rvg Variable: 23 - 10 yr average of 11.5**
- **Global Worgen Variable: 8 - 10 yr average of 4**

<details>
<summary>Click to expand</summary>

image
image

</details>

**Comment by MemeWizard** [comment]
<details>
<summary>1 observer run, 10 years, 1 year check interval, 3 adventurers, 12 sea invaders, 8 packs</summary>

image
image
image

</details>

<details>
<summary>2 observer run, 40 years, 10 years check interval, 6.25 adventurers, 12,25 sea invaders, 1 pack</summary>

image

</details>

<details>
<summary>3 observer run, 10 years, 10 years check interval, 3 adventurers, 12 sea invaders, 0 packs</summary>

image

</details>

<details>
<summary>4 observer run, 100 years, 100 years check interval, 18.3 adventurers, 10.1 sea invaders, 1.1 packs</summary>

image

</details>

<details>
<summary>COD test run, 20 years, 20 years check interval, 5 adventurers, 9.5 sea invaders, 0 packs</summary>

image

</details>

**Comment by MemeWizard** [comment]
[My results](https://docs.google.com/spreadsheets/d/1x2bhrrNHaFXLsKAs3DJbSgTaAypdLpk5VnHnmUnzWWU/edit#gid=0)

TLDR:
10 ravagers / 10 years, average
2 packs / 10 years, average
30-40 adventures / 10 years, average, it's +- vanilla level
